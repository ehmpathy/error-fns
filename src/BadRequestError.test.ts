import { BadRequestError } from './BadRequestError';

describe('BadRequestError', () => {
  it('should produce a helpful, observable error message', () => {
    const error = new BadRequestError('no tires on the vehicle', {
      tires: [],
    });
    expect(error).toMatchSnapshot();
  });
});
