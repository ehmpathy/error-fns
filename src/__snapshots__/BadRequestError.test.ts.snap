// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BadRequestError should produce a helpful, observable error message 1`] = `
[Error: BadRequestError: no tires on the vehicle

{"tires":[]}]
`;
